src/cpu.rs

```diff
@@ -203,14 +187,16 @@ impl Speed for CPU {
 mod tests {
     use super::*;
 
-    #[test]
-    fn render_unit_test() {
-        let mut mock = MockSpeed::new();
-        mock.expect_read_int().return_const(42);
-        mock.expect_read_str().return_const("yflat".to_string());
-
-        // This passes, as expected
-        assert_eq!(mock.read_str("zflat"), "yflat");
-        assert_eq!(mock.read_int("abc"), 42);
-    }
+    // Since the read_int and read_str functions are no longer implemented for speed this test is
+    // pointless I beleive.
+    // #[test]
+    // fn render_unit_test() {
+    //     let mut mock = MockSpeed::new();
+    //     mock.expect_read_int().return_const(42);
+    //     mock.expect_read_str().return_const("yflat".to_string());
+
+    //     // This passes, as expected
+    //     assert_eq!(mock.read_str("zflat"), "yflat");
+    //     assert_eq!(mock.read_int("abc"), 42);
+    // }
```

Comment:
Since read_str and read_int are no longer implemented the test becomes pointless. I don't understand what auto mock does so I don't know if I could reimplement the test with the new functions. @JakeRoggenbuck do you think you can help me with this?

Reply:
@Camerooooon  You can just remove the tests for now

Reply:
read_str are still used right?

Reply:
* and read_int

Reply:
it's just in system now right? you could move the test or delete it for now

Reply:
Those old tests aren't great so I am good with either getting rid of them or just moving them to system

Reply:
Yeah, I'll just remove the test.